app/services/aapb/batch_ingest/csv_item_ingester.rb

```diff
@@ -213,8 +213,8 @@ def set_batch_ingest_id_on_related_asset(work_id, ability)
         def set_admin_data_attributes(admin_data, attributes)
           # add existing admin_data values so they're preserved in the AssetActor
           AdminData.attributes_for_update.each do |admin_attr|
-            next unless attributes.keys.include?(admin_attr.to_s)
-            admin_data.send("#{admin_attr}=", attributes[admin_attr.to_s])
```

Comment:
Why remove the `#to_s`'s here? If I put them back, one of the three failing specs pass

Reply:
it shouldn't be necessary. I found that they start as strings, get converted to symbols, but then in all comparisons (except maybe in a spec somewhere) they are compared as strings. I'm guessing the symbols were used thinking it would be a best practice to store field names as an array of symbols rather than strings.. but if strings are actually what's being used everywhere, it's not a best practice anymore.